src/Uno.Extensions.Reactive.UI/View/FeedView.Refresh.cs

```diff
@@ -45,6 +48,27 @@ public bool CanExecute(object? parameter)
 
 		/// <inheritdoc />
 		public void Execute(object? parameter)
-			=> IsExecuting = _view._subscription?.RequestRefresh() ?? false;
+		{
+			if (_view._subscription is { } subscription)
+			{
+				IsExecuting = true;
```

Comment:
Humm good catch, I was relying on the fact that the `Button` always checks the `CanExecute` but application can still invoke the `Execute` manually. In that case it will reset the flag at the end of the first request, but depending of the feed we can still have a pending request. I'll add a check ...